coderd/prometheusmetrics/aggregator.go

```diff
@@ -22,6 +22,9 @@ const (
 )
 
 const (
+	logPrefix    = "prometheusmetrics: "
+	logPrefixErr = "prometheusmetrics error: "
```

Comment:
Could we use `logger := ma.log.Named("prometheusmetrics"); logger.Debug(...)` instead? I wonder if there's really a need to have the separate err prefix when we already log debug/error levels?

Reply:
Named loggers can be adopted here. Good idea!